### PR TITLE
[termcolor]; Add termcolor

### PR DESCRIPTION
**Pull request by theidexisted**
**Describe the pull request**

- What does your PR fix? Fixes #

Add new library termcolor

- Which triplets are supported/not supported? Have you updated the CI baseline?

support Win and Linux

- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
Yes

### CODE REVIEW

**Comment by prehistoric-penguin** [comment]
CI failed but not caused by my PR:

```


          File path: [ D:/downloads/temp/vtk-vtk-m-f2aa6ad5be1a97e3fb41ef4680ee2c76c3434ac0.tar.gz ]
      Expected hash: [ 2f2a273f74d9a583df9e25a4792440d8d89652fa14b3153f2ea5afbd329b50970e7b9bd68e0ccd036baf5c1f3ad7a8302d95c01dbb30d9a46c045987eebf5370 ]
        Actual hash: [ 35e8a2c0ad6cd3c1f02a71a50d781c89f93909ad27030b406fd69f4fea5c1862c48a6e541fd07562947322c3a69bdfdb54206ae51bb86ef7a710f9e9898e9638 ]

  The file may have been corrupted in transit.  This can be caused by
  proxies.  If you use a proxy, please set the HTTPS_PROXY and HTTP_PROXY
  environment variables to
  "https://user:***@your-proxy-ip-address:port/".

  

Call Stack (most recent call first):
  scripts/cmake/vcpkg_download_distfile.cmake:191 (test_hash)
  scripts/cmake/vcpkg_from_gitlab.cmake:123 (vcpkg_download_distfile)
  ports/vtk-m/portfile.cmake:40 (vcpkg_from_gitlab)
  scripts/ports.cmake:136 (include)


Error: Building package vtk-m:x64-windows failed with: BUILD_FAILED
Elapsed time for package vtk-m:x64-windows: 1.213 s
Starting package 100/104: vtk:x64-windows
Building package vtk[atlmfc,core,paraview,qt,vtkm]:x64-windows...
Error: Building package vtk:x64-windows failed with: CASCADED_DUE_TO_MISSING_DEPENDENCIES
Elapsed time for package vtk:x64-windows: 32 us
```

@NancyLi1013  Could you please take a look?

**Comment by NancyLi1013** [comment]
This is not related with your PR. We have submitted a PR #14884 to fix this.

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

File: ports/termcolor/vcpkg.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "termcolor",
+  "version-string": "1.0.1",
+  "homepage": "https://github.com/ikalnytskyi/termcolor",
+  "description": "Termcolor is a header-only C++ library for printing colored messages to the terminal."
```
Comment (NancyLi1013) [inline ports/termcolor/vcpkg.json:5]:
```suggestion
  "description": "Termcolor is a header-only C++ library for printing colored messages to the terminal.",
  "homepage": "https://github.com/ikalnytskyi/termcolor"
```
Sorry, the order should be like this.

Comment (prehistoric-penguin) [reply ports/termcolor/vcpkg.json]: Thanks, I have verified that the order is right now.

**Review by NancyLi1013 — APPROVED** [review]

**Comment by NancyLi1013** [comment]
The failures caused by `vtk-m` are not related with PR. I will rerun this PR once the regressions are solved.

**Comment by NancyLi1013** [comment]
LGTM now, thanks for your PR @prehistoric-penguin.

**Comment by BillyONeal** [comment]
Thanks!

**Pull request merged**